### PR TITLE
Add test case for shorthand and overriding styles

### PR DESCRIPTION
**Pull request by umr55766**
Test case to better reproduce #441

**Summary :** 

As per https://developer.mozilla.org/en-US/docs/Web/CSS/padding#Syntax

> When three values are specified, the first padding applies to the top, the second to the left and right, the third to the bottom.
<hr>
Working scenario : ✅<br>
Given ➡  "padding : 10px 20px 40px"<br>
Expected ➡  "paddingTop: 10px, paddingLeft: 20px, paddingRight: 20px, paddingBottom: 40px"<br>
Actual ➡  "paddingTop: 10px, paddingLeft: 20px, paddingRight: 20px, paddingBottom: 40px<br>
<hr>
Failing scenario : ❌<br>
Given ➡  "padding : 10px 20px 40px; paddingRight: 30px"<br>
Expected ➡  "paddingTop: 10px, paddingLeft: 20px, paddingRight: 30px, paddingBottom: 40px"<br>
Actual ➡  "paddingTop: 10px, paddingLeft: 20px, paddingRight: '', paddingBottom: 40px" <br>
<hr>

`paddingRight` is getting cleared

Screen Shot 2019-08-16 at 12 40 47 AM

### CODE REVIEW

**Comment by umr55766** [comment]
I am more than willing to provide a fix for that but I am not able to figure where to make code changes.
My guess is I need to make changes in this function https://github.com/snabbdom/snabbdom/blob/1f399d6ddef480d33d0da01907a85d2c2a6427e0/src/modules/style.ts#L34 but somehow when I am adding logs in this function it's not reflecting in output.

I am a beginner for node.js so any help will be much appreciated.

File: src/modules/style.ts [hunk 1]
```diff
@@ -14,8 +14,9 @@ var reflowForced = false;
 function setNextFrame(obj: any, prop: string, val: any): void {
   nextFrame(function() { obj[prop] = val; });
 }
-
+console.log("style.ts called");
 function updateStyle(oldVnode: VNode, vnode: VNode): void {
+  console.log("updateStyle called");
```
Comment (mightyiam) [inline src/modules/style.ts]: @umr55766 seems like there are some forgotten `console.log`s here.

File: src/snabbdom.ts [hunk 2]
```diff
@@ -296,8 +299,10 @@ export function init(modules: Array<Partial<Module>>, domApi?: DOMAPI) {
     }
 
     if (sameVnode(oldVnode, vnode)) {
+      console.log("sameVnode");
       patchVnode(oldVnode, vnode, insertedVnodeQueue);
     } else {
+      console.log("else sameVnode")
```
Comment (mightyiam) [inline src/snabbdom.ts]: @umr55766 seems like there are some forgotten `console.log`s here.

File: test/style.js [hunk 3]
```diff
@@ -44,6 +44,31 @@ describe('style', function() {
...
+  it('updates styles with shorthand', function() {
+    var vnode1 = h('i', {style: {padding: '10px 20px 40px'}});
+
+    elm = patch(vnode0, vnode1).elm;
+    assert.equal(elm.style.paddingTop, '10px');
+    assert.equal(elm.style.paddingLeft, '20px');
+    assert.equal(elm.style.paddingRight, '20px');
+    assert.equal(elm.style.paddingBottom, '40px');
+  });
+  it.only('updates shorthand styles with overriding styles', function() {
```
Comment (nojvek) [inline test/style.js]:
```suggestion
  it('updates shorthand styles with overriding styles', function() {
```

**Comment by mightyiam** [comment]
Thank you, @umr55766. #815 is not desired. So I'll close this PR.

**Pull request closed without merging**